spec/models/jobseeker_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
+
+RSpec.describe Jobseeker, type: :model do
```

Comment:
And do we want to add anything here yet?

Reply:
I don't think we need to add anything just yet, as the model is just empty (other than including Devise functionality, which I think we don't need to test at this low a level, instead once we have a rudimentary "dashboard" we'll build system tests for it).